firebase-installations/src/main/AndroidManifest.xml

```diff
@@ -3,6 +3,8 @@
     package="com.google.firebase.installations">
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
     <uses-permission android:name="android.permission.INTERNET" />
+    <!--Although the *SdkVersion is captured in gradle build files, this is required for non gradle builds-->
```

Comment:
If possible and known, can you please add a reason why Android SDK v14 is required, e.g. because we use some HTTP client that is not available in v13...

Reply:
I have no clue. I will try & findout and update the file. But all Firebase sdks minSDK support is API 14.